psrqpy/search.py

```diff
@@ -1303,6 +1303,36 @@ def derived_equatorial(self):
         self.update(RAJnew, name='RAJ')
         self.update(DECJnew, name='DECJ')
 
+        ELONG_ERR = self.catalogue['ELONG_ERR']
```

Comment:
I think you'll need to make this line:

```python
if 'ELONG_ERR' in self.columns:
    ELONG_ERR = self.catalogue['ELONG_ERR']
else:
    ELONG_ERR = np.full(self.catalogue_len, np.nan)
```

Reply:
Thanks for the suggestion. I hadn't come up with a solution to deal with missing values, but this would do the trick.